### PR TITLE
Removed rigid dict

### PR DESCRIPTION
**Pull request by miggol**
Replaced it with container classes

### CODE REVIEW

**Comment by miggol** [comment]
Oh, and also this fixes the PDF link bug. If that wasn't obvious

File: reviews/templatetags/documents_list.py [hunk 1]
```diff
@@ -245,7 +244,7 @@ def documents_list_v2(review, user):
 
     for d in qs:
         items = []
-        files = [
+        potential_files = [
```
Comment (tymees) [inline reviews/templatetags/documents_list.py:247]: Why are they 'potential'?

Comment (tymees) [reply reviews/templatetags/documents_list.py:247]: nvm, i get it now

**Review by tymees — APPROVED** [review]
Looks good! I did spot some PEP8 problems (too much whitespace between classes/functions), but that's minor

Feel free to merge whenever you want

**Pull request merged**